### ISSUE TITLE
`.DayPicker-wrapper` `tabIndex` and `aria-label`

### ISSUE THREAD

**Issue by catamphetamine** [gpbl/react-day-picker#848]
By default the "wrapper" div is focusable via Tab because it has `tabIndex = 0`.
And also the first day of the month is focusable due to it also having `tabIndex = 0`.

If `tabIndex = -1` property is passed then neither the wrapper is focusable via Tab nor the first day of the month is, because both of them get `tabIndex = -1`.

A finer level of granularity would be more appropriate.
For example, having two separate properties for "tab index".
Or maybe, better having `tabIndex = 0` on the first day of the month always because otherwise keyboard-only users would be unable to choose a date.

One more thing is that the `wrapper` div doesn't have any `aria-label` property.
So if it has `tabIndex = 0` and is focused then the narrator is silent and the user will be confused in such case and won't know where the focus is.
An `aria-label` consisting of active month and year would be appropriate.

Also, there's no `tabIndex` setting for prev/next month buttons.
You might add a separate property for that.

**Comment by rootlinux2** [comment]
To make this calendar more accessible I think you need to add ARIA-* props to the wrapper because as a developer I need to control all these features.

**Comment by aitchkhan** [comment]
Well, it varies from case to case.
But IMHO, `tabIndex = 0` should be either on the day passed in `selectedDays` props or `today`.
